### PR TITLE
Tableau vs Power BI

### LINKED ISSUE #450: Tableau Vs. Power BI
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.
Microsoft Power BI and Tableau are the two front-runners in the data visualization and business intelligence (BI) software industry. Both offer advanced reporting and visualization capabilities. Each has its own strengths and weaknesses when it comes to use in business intelligence. 
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
1) Overview of Tableau
2) Overview of Power BI
3) Key Differences 
       a) Integration
       b) data visualizations and dashboard
       c) data connectivity 
       d)Analytics 
       e)Data Sources
4) Which is better for business intelligence?  

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
https://www.smetricinsights.com/wp-content/uploads/2019/05/Tableau-VS-QlikView-VS-Power-BI-2019-Update.pdf
https://technologyadvice.com/blog/information-technology/power-bi-vs-tableau/#:~:text=Power%20BI%20comes%20at%20a,intelligence%20to%20enhance%20their%20analytics.
https://www.selecthub.com/business-intelligence/tableau-vs-qlikview-vs-microsoft-power-bi/

### PR DESCRIPTION
**Pull request by EKahuha**
This fixes #450

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      

#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tool:** [Grammarly](https://grammarly.com)
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/tableau-vs-power-bi/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Power BI&#39;s analytics interface closely resembles Excel. This assures recognizable functionality and an easy-to-learn Power BI experience. Like with Tableau, you can perform complex data analysis using Microsoft Power BI. An analysis with Power BI can include premise-based integration points and data sources. Power BI Desktop and Power BI Suite contain data discovery and interactive dashboards.
+
+#### Data Sources and Data Connectivity
+
+A key strength of Tableau is its ability to support many data connectors. These include the could option, [online analytical processing (OLAP)](https://olap.com/olap-definition/), as well as big data options like Hadoop and NoSQL. When you add data from multiple sources, Tableau will determine the relationship automatically. You can create or modify data links manually based on your preference. You can connect to a distinct data warehouse if you use Tableau.
```
Comment (lalith1403) [inline articles/tableau-vs-power-bi/index.md:39]: It's not clear what you mean by could option. Probably a typo?

File: articles/tableau-vs-power-bi/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### An Overview of Tableau
+
+[Tableau](https://www.tableau.com/) is an analytics tool that helps to combine and integrate data from many sources into visualizations. Users can access this data in a single desktop environment with the use of Tableau Desktop. Alternatively, the data can be accessed through a shared dashboard. Tableau&#39;s user-friendly interface allows users to analyze data easily, create visualizations and reports, and share insights.
+
+Basically, Tableau Desktop acts as a report development tool. You can use this tool to create reports and format your data into visualizations. You can then organize visualizations into a dashboard for personal use or share the same through Tableau Server. The Tableau Server portal is secure, and you can share your dashboard with others from any mobile device or browser.
+
+### An Overview of Power BI
```
Comment (lalith1403) [inline articles/tableau-vs-power-bi/index.md:9]: Add a section indicating the various parameters this article will be comparing the software on. That will give a clean and polished introduction, to begin with.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
@EKahuha Great job! I loved reading the software-review article. A couple of minor changes and the article should be good to go for the next round of review. Let me know once updated.

**Comment by EKahuha** [comment]
Hi @lalith1403 
File updated successfully.

**Comment by hectorkambow** [comment]
plagiarism check done

**Review by lalith1403 — APPROVED** [review]

**Pull request merged**